gc/policy/policy.go

```diff
@@ -0,0 +1,217 @@
+package policy
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/containerd/containerd/log"
+	"github.com/containerd/containerd/metadata"
+	"github.com/containerd/containerd/plugin"
+)
+
+type config struct {
+	PauseThreshold float64 `toml:"pause_threshold"`
+
+	ScheduleOnDelete  bool `toml:"on_delete"`
+	MutationThreshold int  `toml:"mutation_threshold"`
+
+	ScheduleDelayMs int `toml:"schedule_delay_ms"`
+	StartupDelayMs  int `toml:"startup_delay_ms"`
+}
+
+func init() {
+	plugin.Register(&plugin.Registration{
+		Type: plugin.GCPlugin,
+		ID:   "policy",
+		Requires: []plugin.Type{
+			plugin.MetadataPlugin,
+		},
+		Config: &config{
```

Comment:
Can we read this from config.toml?

Reply:
What do you mean?

This does need to be better documented, bother for the individual fields and in some configuration document. I had trouble finding what the actual toml key was supposed to be to configure this section (it was `plugings.policy`).

Reply:
Perhaps I misunderstood you, this only sets the defaults, the toml will overwrite these values.